### PR TITLE
S11 delete event button

### PR DESCRIPTION
**Pull request by ramsey-merdassi**

### CODE REVIEW

File: src/Controllers/API/AddEventController.php [hunk 1]
```diff
@@ -38,6 +38,7 @@ public function __invoke(Request $request, Response $response, array $args): Res
             ) {
                 $newEvent = EventSanitiser::sanitise($newEvent);
                 $result = $this->eventModel->addEvent($newEvent);
+//                $this->eventModel->updateEvents();
```
Comment (joseph-padfield) [inline src/Controllers/API/AddEventController.php:41]: what's this? it's currently commented out

**Review by joseph-padfield — DISMISSED** [review]
Looks great, just one change

File: src/Models/EventModel.php [hunk 2]
```diff
@@ -189,10 +197,10 @@ public function getEventsByCategoryId(string $categoryId, int $previousMonths =
     public function getUpcomingEventsByCategoryIdAndSearch(?string $categoryId = null, ?string $searchQuery = ''): array
     {
         $sql = 'SELECT `events`.`id`, `events`.`name`, `events`.`category`, 
-        `event_categories`.`name` AS `category_name`, `location`, `date`, `start_time`,`end_time`, `notes`
+        `event_categories`.`name` AS `category_name`, `location`, `date`, `start_time`,`end_time`, `notes`, `deleted`
```
Comment (matt-ioa) [inline src/Models/EventModel.php:200]: I don't think you need to select `deleted` here, it's just enough to have `WHERE deleted = 0` later

File: src/Models/EventModel.php [hunk 3]
```diff
@@ -214,10 +222,10 @@ public function getUpcomingEventsByCategoryIdAndSearch(?string $categoryId = nul
     public function getPastEventsByCategoryIdAndSearch(?string $categoryId = null, ?string $searchQuery = ''): array
     {
         $sql = 'SELECT `events`.`id`, `events`.`name`, `events`.`category`, 
-        `event_categories`.`name` AS `category_name`, `location`, `date`, `start_time`,`end_time`, `notes`
+        `event_categories`.`name` AS `category_name`, `location`, `date`, `start_time`,`end_time`, `notes`, `deleted`
```
Comment (matt-ioa) [inline src/Models/EventModel.php:225]: Same here

**Review by matt-ioa — CHANGES_REQUESTED** [review]
Just a couple of comments

**Review by matt-ioa — APPROVED** [review]

**Review by matt-ioa — APPROVED** [review]

**Pull request merged**